### ISSUE TITLE
[ElasticDL Client] ElasticDL Job submission template

### ISSUE THREAD

**Issue by yuyicg** [sql-machine-learning/elasticdl#208]
the template contains some info such as:
* nn.Module
* loss function
* hyper-parameter
* training/validation data
* data preprocessing/augmentation func (optional)
